### PR TITLE
Add livestream gui

### LINKED ISSUE #118: [FEATURE] Make sure we can stream our GUI to the MATE people
### Is your feature request related to a problem? Please describe

We have to stream a view of our GUI to the judges (?) during competition.

### Describe the solution you'd like

Figure out how to do this. Read the rules. HDMI capture? They have an HDMI cable?

### PR DESCRIPTION
**Pull request by NoahMollerstuen**
# Pull Request

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds a gui to be displayed on the MATE livestream. Use the new `competition_launch.py` launch file in `surface_main`.

## Related Tickets & Documents

- Related Issue #
- Closes #118

### CODE REVIEW

File: src/surface/surface_main/launch/competition_launch.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from ament_index_python.packages import get_package_share_directory
+from launch.launch_description import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+
+def generate_launch_description() -> LaunchDescription:
+
+    surface_path: str = get_package_share_directory('surface_main')
+    gui_path: str = get_package_share_directory('gui')
```
Comment (InvincibleRMC) [inline src/surface/surface_main/launch/competition_launch.py:12]:
```suggestion
    gui_path = get_package_share_directory('gui')
```

File: src/surface/surface_main/launch/competition_launch.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+from ament_index_python.packages import get_package_share_directory
+from launch.launch_description import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+
+def generate_launch_description() -> LaunchDescription:
+
+    surface_path: str = get_package_share_directory('surface_main')
```
Comment (InvincibleRMC) [inline src/surface/surface_main/launch/competition_launch.py:11]:
```suggestion
    surface_path = get_package_share_directory('surface_main')
```

**Review by InvincibleRMC — CHANGES_REQUESTED** [review]
Add back switching depth camera.

**Comment by NoahMollerstuen** [comment]
> Add back switching depth camera.

I took it out intentionally, I don't think we're planning to have a depth cam anymore. Unless we want it for something other than modelling.

**Comment by InvincibleRMC** [comment]
> > Add back switching depth camera.
> 
> I took it out intentionally, I don't think we're planning to have a depth cam anymore. Unless we want it for something other than modelling.

We are trying to add it back so we can have a back camera just in case.

**Comment by NoahMollerstuen** [comment]
> > > Add back switching depth camera.
> > 
> > 
> > I took it out intentionally, I don't think we're planning to have a depth cam anymore. Unless we want it for something other than modelling.
> 
> We are trying to add it back so we can have a back camera just in case.

Aight I added it back

**Review by InvincibleRMC — APPROVED** [review]
lgtm.

**Pull request merged**